yt/utilities/grid_data_format/conversion/conversion_athena.py

```diff
@@ -130,8 +130,8 @@ def read_and_write_index(self, basename, ddn, gdf_name):
                 grid["dimensions"][grid["dimensions"] == 0] = 1
             if np.prod(grid["dimensions"]) != grid["ncells"]:
```

Comment:
don't want a walrus here too??? :)

Reply:
if I were to write this from scratch maybe, but here I like that it's the exact same conditions than 3 lines before.